auto sync 2022-03-03

nunnatsa opened the pull request:
auto sync upstream 2022-03-03

JoelSpeed commented:
This doesn't actually do anything for this provider presently, but was automatically created by the script. Do you think you'll be able to up date to a newer MAO at any point in the near future or are you still stuck on the older versions?

rgolangh commented:
/lgtm
/approve

rgolangh commented:
/hold

rgolangh commented:
@JoelSpeed where is the vendor update? you didn't run make vendor on purpose?

JoelSpeed commented:
> @JoelSpeed where is the vendor update? you didn't run make vendor on purpose?

@rgolangh This update was created by a script that didn't take into account the fact you have a replace statement at the moment. I've just pushed an update that includes removing the replacement, not sure if this will work or not. May have to spend a bit of time ironing out some changes

JoelSpeed commented:
I've detached this from the bug for now as I need to get all of the other providers verified for the 4.7 release, we can keep this open or close as you prefer based on my previous comment

chenyosef commented:
@JoelSpeed I believe this was handled here https://github.com/openshift/cluster-api-provider-kubevirt/pull/23 ?

JoelSpeed commented:
Yep, thanks for handling that!

nirarg committed:
Change OWNERS file

nunnatsa committed:
Automatic sync with upstream

This PR adds a new github action to daily sync with upstream. It checks if there are new changes in upstream, and if there are, it creates a PR to sync.

The new PR is created by hard reseting to the upstream main branch, then restoring the downstream modified files.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>
